pkg/flow/adapter/transformation/adapter.go

```diff
@@ -81,59 +81,42 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 	trnContext, trnData := []v1alpha1.Transform{}, []v1alpha1.Transform{}
 	err := json.Unmarshal([]byte(env.TransformationContext), &trnContext)
 	if err != nil {
-		log.Fatalf("Cannot unmarshal Context Transformation variable: %v", err)
+		log.Fatalf("Cannot unmarshal context transformation env variable: %v", err)
```

Comment:
While we're at fatal'ing things, any chance we can use Knative's logger instead of Go's stdlib `log` package, like everywhere else in TriggerMesh?

https://github.com/triggermesh/triggermesh/blob/f88cfe4e78ea44bb2ca8290cad8f2cf10d9bb059/pkg/sources/adapter/googlecloudpubsubsource/adapter.go#L73

...

https://github.com/triggermesh/triggermesh/blob/f88cfe4e78ea44bb2ca8290cad8f2cf10d9bb059/pkg/sources/adapter/googlecloudpubsubsource/adapter.go#L81

I'm fine if you prefer to do it in a separate PR.

Reply:
There are a couple of other places where it needs to be changed, so I'd make it in a separate PR
https://github.com/triggermesh/triggermesh/issues/780